### PR TITLE
The counter is now hidden when viewing a recipe, the display of hard/…

### PR DESCRIPTION
**Pull request by Zakari-B**
…soft cocktails in the cocktail list can be toggled, a button has been added as a placeholder for the login, a few typos and bugs fixed

### CODE REVIEW

File: frontend/src/pages/Home.jsx [hunk 1]
```diff
@@ -218,6 +219,15 @@ export default function Home() {
...
   };
+
+  const counterDivToggle = () => {
+    if (displayRecipe === false) {
+      buttonDiv.style.display = "none";
+    } else {
+      buttonDiv.style.display = "flex";
+    }
+  };
+
```
Comment (lchastanet) [inline frontend/src/pages/Home.jsx:230]: This cannot be done on re-render ?

File: frontend/src/pages/Home.jsx [hunk 2]
```diff
@@ -262,6 +272,7 @@ export default function Home() {
               <button
                 className="select-button"
                 type="button"
+                // onClick={() => counterDivToggle()}
```
Comment (lchastanet) [inline frontend/src/pages/Home.jsx:275]: Commented code here

**Review by lchastanet — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Home.jsx (hunk 1 above)
Comment (Zakari-B) [reply frontend/src/pages/Home.jsx:230]: Not with the way things are done. The states do not empty on dismounting a component but the DOM does, breaking the search function.

File: frontend/src/pages/Home.jsx (hunk 2 above)
Comment (Zakari-B) [reply frontend/src/pages/Home.jsx]: Yes, I missed this one, it has been deleted.

File: frontend/src/pages/Home.jsx (hunk 1 above)
Comment (Zakari-B) [reply frontend/src/pages/Home.jsx:230]: I'll need to try a few things to be sure but it should be possible.

File: frontend/src/pages/Home.jsx (hunk 2 above)
Comment (lchastanet) [reply frontend/src/pages/Home.jsx]: Ok, great, keep me posted !

File: frontend/src/pages/Home.jsx (hunk 1 above)
Comment (Zakari-B) [reply frontend/src/pages/Home.jsx]: Changed counterDivToggle and replaced it with a state to be more in line with how react works.

**Review by lchastanet — APPROVED** [review]
Ok 👌🏻

**Pull request merged**